modMcf/src/org/aion/mcf/db/TransformedCodeInfo.java

```diff
@@ -0,0 +1,43 @@
+package org.aion.mcf.db;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.aion.util.types.ByteArrayWrapper;
+
+public class TransformedCodeInfo {
+
+    // Key for this map is the codeHash, the value is a map which has AVM version as its key and transformed code as its value.
+    public Map<ByteArrayWrapper, Map<Integer, ByteArrayWrapper>> transformedCodeMap;
```

Comment:
The values for the inner map should be `byte[]` since they are not used as keys, the wrapper would only decrease performance.